### DISCUSSION TITLE
Reformulação da PythOnRio

### DISCUSSION THREAD

**Discussion by jamesperes** [pythonrio/organizacao#15] [Ideas]
Fala pessoal, Gostaria de sugerir uma reformulação do nosso querido evento da PythOnRio nos seguintes tópicos .
- Pedir Apoio a APyB
  - a Bia já viu que podemos pedir uma verba por ano junto com a [APyB](https://github.com/apyb/comunidade/discussions/29)  está aqui o link.
- Patrocínio anual de empresas
  - Pedir um patrocínio onde durante todos os eventos que forem cuidados pela PythOnRio apareça a marca da empresa. Com isso pode ajudar a custear os eventos. 
- Calendário diversificado
   - Calendário Baseado em Médios eventos a cada 3 meses e nesses intervalos eventos com o objetivo de unir pessoas.Ex de calendario:

> 1 - Jan - BrejaTech RJ - Evento no Centro do Rio e meio da semana para conhecer Pessoas de Tecnologia.
> 2 - Fev - Hora Extra Barra -  Evento na Barra e meio da semana para conhecer Pessoas de Tecnologia.
> 3 - Mar - Just Python - Evento onde todos as palestras devem usar somente libs do python
> 4 - Abr - BrejaTech RJ - Evento no Centro do Rio e meio da semana para conhecer Pessoas de Tecnologia.
> 5 - Mai - Hora Extra Barra -  Evento na Barra e meio da semana para conhecer Pessoas de Tecnologia.
> 6 - Jun - CaipiraRJ - Evento voltado para Programação em geral 
> 7 - Jul - BrejaTech RJ - Evento no Centro do Rio e meio da semana para conhecer Pessoas de Tecnologia.
> 8 - Ago - Django Girls Rio.
> 9 - Set - Flask Conf Rio 
> 10 - Out - Python Brasil
> 11 - Nov - BrejaTech RJ - Evento no Centro do Rio e meio da semana para conhecer Pessoas de Tecnologia.
> 12 - Dez - Sprints Open Source - Um dia inteiro de Codigo

- Apoio das Universidades
   - Buscar apoio de Universadades para eventos e programar o evento para junto de uma semana de programação do lugar.
- Evento de sprints
  - Um evento totalmente voltado para fazer codigo

**Comment by ffabiorj** [comment]
Boa ideia.

**Comment by biancarosa** [comment]
@jamesperes acho as ideias boas! acho que pra ganhar cadencia podemos focar em eventos a cada 3 meses e não mensais porque tem sido dificil organizar com essa frequencia, mas fora isso to de acordo com tudo!

**Comment by this-lincoln** [comment]
Achei a ideia ótima @jamesperes , dessa forma o pessoal consegue fazer também os eventos locais entre as PyRio. Imagina, um Hora Extra por mês e a PyRio a cada 3 meses? Seria lindo.
